spec.bs

```diff
@@ -358,19 +666,67 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 <div algorithm>
   The <dfn method for=Fence>reportEvent(<var ignore>event</var>)</dfn> method steps are:
 
-  1. <span class=XXX>Fill this out!</span>
+  1. Let *instance* be this [=traversable navigable=]'s [=fenced frame config instance=].
```

Comment:
We should be able to more formally access the traversable navigable via `Let |instance| be [=this=]'s [=relevant global object=]'s [=associated Document=]'s [=Document/node navigable=]'s [=navigable/traversable navigable=]` I think (here and any other places in this PR).

Reply:
It gave me an error with node navigable in Document, so I changed it to:

`[=this=]'s [=relevant global object=]'s [=associated Document=]'s [=node navigable=]'s [=navigable/traversable navigable=]`